components/org.wso2.carbon.identity.scim2.provider/src/main/java/org/wso2/carbon/identity/scim2/provider/resources/BulkResource.java

```diff
@@ -66,7 +66,7 @@ public Response createUser(@HeaderParam(SCIMProviderConstants.AUTHORIZATION) Str
             encoder = identitySCIMManager.getEncoder();
 
             // obtain the user store manager
-            UserManager userManager = IdentitySCIMManager.getInstance().getUserManager();
+            UserManager userManager = identitySCIMManager.getUserManager();
```

Comment:
why was this change required?

Reply:
To avoid instantiating IdentitySCIMManager twice in the same method